src/inputType/PanInput.ts

```diff
@@ -266,8 +266,10 @@ export class PanInput implements IInputType {
 			]);
 		const prevent = offset.some(v => v !== 0);
 		if (prevent) {
-			event.srcEvent.preventDefault();
-			event.srcEvent.stopPropagation();
+			const srcEvent = event.srcEvent;
+
```

Comment:
@WoodNeck 
I'm not sure, but sometimes cancelable doesn't exist.

```js
srcEvent.cancelable !== false
```

Reply:
@WoodNeck 

If cancelable is false, are you trying to stop all drag events? 


I don't think a `hold` and `change` event should happen.

Reply:
As you said, some of the old browsers don't support `cancelable` property.
But in that case, I think it should be
```js
srcEvent.cancelable === true
```
Anyway, I applied your review. Thanks!